squid/tox.ini

```diff
@@ -1,21 +1,17 @@
 [tox]
 minversion = 2.0
 basepython = py27
-envlist = unit, squid-latest, flake8
+envlist =
+  unit
```

Comment:
I think it'd be useful to run python3 against unit tests as well. Typically I think unit tests cover some utility functions that aren't necessarily hit by the integration tests, so ensuring those work with Python3 would be helpful in my opinion.